frontend/src/host_orchestrator/orchestrator/controller.go

```diff
@@ -58,6 +58,7 @@ func (c *Controller) AddRoutes(router *mux.Router) {
 	router.Handle("/cvds",
 		httpHandler(newCreateCVDHandler(c.Config, c.OperationManager, c.UserArtifactsManager))).Methods("POST")
 	router.Handle("/cvds", httpHandler(&listCVDsHandler{Config: c.Config})).Methods("GET")
+	router.Handle("/cvdgroups", httpHandler(&listCVDGroupsHandler{Config: c.Config})).Methods("GET")
```

Comment:
cvd groups have no significant meaning on their own. Either add an extra `group` field to the cvd descriptions we return or make the `cvds` resource a subresource of groups.

I think the extra fields in `CVD` makes the most sense for now: It's the simplest of the two while returning the same amount of information while not tying us to a specific solution prematurely (you can continue to return the group name as part of the CVD object even if you later decide to add more endpoints and functionality).

Similarly, you can always specify the group name when creating a new cvd instead of creating a group first and then adding cvds (not even sure this is supported yet).

Reply:
added the extra fields to CVD.